plat/nvidia/tegra/common/tegra_gic.c

```diff
@@ -0,0 +1,95 @@
+/*
```

Comment:
AFAICS, you are using the raw GIC accessors in `gic_v2.c` which take the base address as an argument. This allows you to bypass the ARM GIC driver. However, in `runtime_exceptions.S` we have calls to some platform provided GIC functions e.g. plat_ic_get_pending_interrupt_type(). The weak definitions of these functions in `plat/common/plat_gic.c` fall into the corresponding functions in the ARM GIC driver e.g. `arm_gic_get_pending_interrupt_type()`. Since the ARM GIC driver is not initialised on the TEGRA port, the base addresses in this routines will be NULL. This will lead to trouble on both the release and debug builds of BL3_1 when interrupts are routed to EL3. 

Are you not seeing issues with the TLKD or am I missing something?